src/incubator/TextField/index.tsx

```diff
@@ -83,9 +84,13 @@ type TextFieldProps = MarginModifiers &
      * Container style of the whole component
      */
     containerStyle?: ViewStyle;
+    /**
+     * Predefined preset to use for styling the field
+     */
+    preset?: 'default' | undefined;
```

Comment:
Is the type could be optional boolean? 
(`useDefaultPreset` for example)

Reply:
Actually the idea was to add more presets in the future, so the `preset` prop was meant to be used as an enum. 
For now I just added the `default` preset so it feels redundant..

Reply:
Yeah that's what I thought it meant to be, merging